src/test/suite/command-line.test.ts

```diff
@@ -25,30 +32,41 @@ describe("Command line", () => {
         await vscode.window.showTextDocument(doc, vscode.ViewColumn.One);
         await wait();
 
-        await sendVSCodeKeys("/");
+        await sendVSCodeKeysAtomic("/1");
+        await wait(1000);
         await vscode.commands.executeCommand("vscode-neovim.match-cursor-search-cmdline");
+        await wait(100);
         await vscode.commands.executeCommand("vscode-neovim.match-cursor-search-cmdline");
-        await sendVSCodeKeys("\n");
-        assert.equal(await client.commandOutput("echo getreg('/')"), "1a");
+        await wait(100);
+        await vscode.commands.executeCommand("vscode-neovim.commit-cmdline");
+        assert.equal(await client.commandOutput("echo getreg('/')"), "1ab");
```

Comment:
I believe, when in search, <C-l> is supposed to match starting with the character under the cursor if nothing is provided.

That's what this test was supposed to check.

Reply:
Yeah I'm not sure, I tried in terminal nvim and got the same behavior.

Reply:
Do you mean that the original test did work for you?